src/model/FaceliftProperty.h

```diff
@@ -51,6 +51,9 @@ class FaceliftModelLib_EXPORT PropertyBase
 
     virtual ~PropertyBase();
 
+    PropertyBase(PropertyBase const &) = delete;
```

Comment:
Could you also take care of &&? (rule of five)

Reply:
done